### PR TITLE
Refactor to MYMorise Test

### PR DESCRIPTION
**Pull request by choonx99**

### CODE REVIEW

File: src/test/data/JsonExpenseListStorageTest/invalidExpenseExpenseList.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "expenses": [ {
+    "name": "Expense with invalid name field: @!asda24",
```
Comment (muhammadaljunied) [inline src/test/data/JsonExpenseListStorageTest/invalidExpenseExpenseList.json:3]: Is @ ! and numbers not permissible in the name?

Comment (choonx99) [reply src/test/data/JsonExpenseListStorageTest/invalidExpenseExpenseList.json:3]: it says names only allow alphanumeric characters and spaces, so i dont think special characters are allowed?

**Review by muhammadaljunied — APPROVED** [review]

File: src/test/data/JsonExpenseListStorageTest/invalidExpenseExpenseList.json (hunk 1 above)
Comment (Cary-Xx) [reply src/test/data/JsonExpenseListStorageTest/invalidExpenseExpenseList.json:3]: should we allow special char?

File: src/test/data/JsonExpenseListStorageTest/invalidAndValidExpenseExpenseList.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "expenses": [ {
+    "name": "Valid Expense",
+    "amount": "$23.50",
+    "date": "13/10/2019",
+    "tagged": [ "education" ]
+  }, {
+    "name": "Expense with Invalid amount",
+    "amount": "$94asd44",
+    "date": "hans@example.com",
```
Comment (Cary-Xx) [inline src/test/data/JsonExpenseListStorageTest/invalidAndValidExpenseExpenseList.json]: change to a valid date

File: src/test/java/seedu/address/commons/util/AppUtilTest.java [hunk 3]
```diff
@@ -9,7 +9,7 @@
 
     @Test
     public void getImage_exitingImage() {
-        assertNotNull(AppUtil.getImage("/images/address_book_32.png"));
+        assertNotNull(AppUtil.getImage("/images/expense_list_32.png"));
```
Comment (Cary-Xx) [inline src/test/java/seedu/address/commons/util/AppUtilTest.java:12]: Rename the png under resources

File: src/test/java/seedu/address/testutil/EditExpenseDescriptorBuilder.java [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    public EditExpenseDescriptorBuilder() {
+        descriptor = new EditExpenseDescriptor();
+    }
+
+    public EditExpenseDescriptorBuilder(EditExpenseDescriptor descriptor) {
+        this.descriptor = new EditExpenseDescriptor(descriptor);
+    }
+
+    /**
+     * Returns an {@code EditExpenseDescriptor} with fields containing {@code expense}'s details
```
Comment (Cary-Xx) [inline src/test/java/seedu/address/testutil/EditExpenseDescriptorBuilder.java]: @code EditExpenseDescriptorBuilder

File: src/test/java/seedu/address/testutil/ExpenseBuilder.java [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+import seedu.address.model.expense.Name;
+import seedu.address.model.expense.Expense;
+import seedu.address.model.expense.Amount;
+import seedu.address.model.expense.Date;
+import seedu.address.model.tag.Tag;
+import seedu.address.model.util.SampleDataUtil;
+
+/**
+ * A utility class to help with building Expense objects.
+ */
```
Comment (Cary-Xx) [inline src/test/java/seedu/address/testutil/ExpenseBuilder.java]: add ` * Example usage: <br>
 *     {@code ExpenseList ab = new ExpenseListBuilder().withName("Coffee").build();}`

File: src/test/java/seedu/address/testutil/ExpenseBuilder.java [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+public class ExpenseBuilder {
```
Comment (Cary-Xx) [inline src/test/java/seedu/address/testutil/ExpenseBuilder.java]: Rename to ExpenseListBuilder

File: src/test/java/seedu/address/testutil/ExpenseBuilder.java [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+    public static final String DEFAULT_NAME = "coffee";
+    public static final String DEFAULT_AMOUNT = "$23.50";
+    public static final String DEFAULT_DATE = "13/10/2019";
+
+    private Name name;
+    private Amount amount;
+    private Date date;
+    private Set<Tag> tags;
+
+    public ExpenseBuilder() {
```
Comment (Cary-Xx) [inline src/test/java/seedu/address/testutil/ExpenseBuilder.java]: should construct an expenseListBuilder with empty fields, see the usage above

File: src/test/java/seedu/address/testutil/ExpenseListBuilder.java [hunk 8]
```diff
@@ -0,0 +1,34 @@
+package seedu.address.testutil;
+
+import seedu.address.model.ExpenseList;
+import seedu.address.model.expense.Expense;
+
+/**
+ * A utility class to help with building Addressbook objects.
+ * Example usage: <br>
+ *     {@code ExpenseList ab = new AddressBookBuilder().withExpense("John", "Doe").build();}
+ */
```
Comment (Cary-Xx) [inline src/test/java/seedu/address/testutil/ExpenseListBuilder.java:10]: uniform the naming

File: src/test/java/seedu/address/testutil/TypicalExpenses.java [hunk 9]
```diff
@@ -0,0 +1,72 @@
+package seedu.address.testutil;
```
Comment (Cary-Xx) [inline src/test/java/seedu/address/testutil/TypicalExpenses.java:1]: data in `typicalExpensesExpenseList.json` should be the same with this

File: src/main/resources/view/DarkTheme.css [hunk 10]
```diff
@@ -328,7 +328,7 @@
     -fx-text-fill: white;
 }
 
-#filterField, #expenseListPanel, #personWebpage {
+#filterField, #expenseListPanel, #expenseWebpage {
```
Comment (Cary-Xx) [inline src/main/resources/view/DarkTheme.css]: revert this change and also expenseListPanel
